src/sidechaindb.cpp

```diff
@@ -308,179 +360,238 @@ bool SidechainDB::Update(int nHeight, const uint256& hashBlock, const std::vecto
             }
         }
     }
-    hashBlockLastSeen = hashBlock;
-    return true;
-}
-
-uint256 SidechainDB::GetHashBlockLastSeen()
-{
-    return hashBlockLastSeen;
-}
 
-std::multimap<uint256, int> SidechainDB::GetLinkingData() const
-{
-    return mapBMMLD;
-}
+    // Scan for new WT^(s) and start tracking them
+    // TODO
+    // SidechainDB::AddWTJoin
+    for (const CTxOut& out : vout) {
+        const CScript& scriptPubKey = out.scriptPubKey;
+        if (scriptPubKey.IsWTHashCommit()) {
+            // Get WT^ hash from script
```

Comment:
So i understand we are getting `WT^`'s hash, but where do we get the full `WT^` transaction at? A hash doesn't do us much good if we don't have the entire transaction? So shouldn't this be the entire transaction embedded in the output?